scripts/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Inputs: $1 = channel (dev, staging, rc)
+# Outputs: JSON string "{ "version": "v1.5.0", "link": "pear://key", "success": "true" }"
+
+channel=$1
+
+devkey=TODO
+stagekey=TODO
+
+if [ $channel == "dev" ]; then
+  ./pear.dev sidecar shutdown
+
+  npm i
+
+  rm -fr by-arch
+  npm run archdump
+
+  rm -rf ./node_modules
+  npm i --omit=dev
+  npm run prune
+
+  ./pear.dev sidecar &
+  ./pear.dev seed dev &
```

Comment:
not forking is allowed in the build script

Reply:
removed